src/analyzing/Constant_propagation.mli

```diff
@@ -6,6 +6,20 @@ val constant_propagation_and_evaluate_literal :
   ?lang:Lang.t -> AST_generic.expr -> AST_generic.svalue option
 (** Partially evaluate a Generic expression. *)
 
+type env
+
+type propagate_basic_visitor_funcs = {
```

Comment:
Maybe add some comments to clarify what we are exporting to be used by Pro.

Reply:
Done.